src/xenia/cpu/testing/pack_test.cc

```diff
@@ -51,7 +51,7 @@ TEST_CASE("PACK_FLOAT16_2", "[instr]") {
       },
       [](PPCContext* ctx) {
         auto result = ctx->v[3];
-        REQUIRE(result == vec128i(0, 0, 0, 0x7FFFFFFF));
```

Comment:
Ok I see these tests fail on master. But are you sure the test case is at fault here? (This comment counts for all changes in this commit)

Reply:
I was using win32 as a working reference.
All the changes are done to match what is seen in win32.
I am not sure if these are the right outputs because these tests are pretty old.